packages/patternfly-4/react-core/src/components/Card/Card.tsx

```diff
@@ -21,18 +25,23 @@ export const Card: React.FunctionComponent<CardProps> = ({
   component = 'article',
   isHoverable = false,
   isCompact = false,
+  isSelectable = false,
+  isSelected = false,
   ...props
 }: CardProps) => {
   const Component = component as any;
   return (
     <Component
+      {...props}
       className={css(
         styles.card,
         isHoverable && styles.modifiers.hoverable,
         isCompact && styles.modifiers.compact,
+        isSelectable && styles.modifiers.selectable,
+        isSelected && isSelectable && styles.modifiers.selected,
         className
       )}
-      {...props}
+      tabIndex={isSelectable && isSelected ? '0' : undefined}
```

Comment:
The selectable card is what needs `tabindex="0"`, whether it's selected or not.

```suggestion
      tabIndex={isSelectable ? '0' : undefined}
```